config/initializers/flood_risk_engine.rb

```diff
@@ -4,13 +4,13 @@
 
 FloodRiskEngine.configure do |config|
   # Last email cache config
-  config.use_last_email_cache = ENV["USE_LAST_EMAIL_CACHE"] || "false"
+  # FIX: config.use_last_email_cache = ENV["USE_LAST_EMAIL_CACHE"] || "false"
```

Comment:
Do we need to add a new ticket to sort this out?

Reply:
I must have disabled this in the upgrade. Restored, and seems to be working (i.e. the app is booting up)